interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is like a math equation that has variables and that output a soulution 
 
-  Researched answer:
+  Researched answer: A function is a piece of code that runs and returns a value.
 
 
 
 2. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: a parameter is almost like a equation that you want to have sovled, and an agurment are the variables for the parameters
```

Comment:
More closely pramater is like "x" in "x + 3 = z" we need to give x as an argument to see what z is.